hack/release-manifests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ex
+
+SELF=$( realpath $0 )
+BASEPATH=$( dirname $SELF )
+
+TAG="${1:-v0.0.0}"
+VERSION=$( ${BASEPATH}/../build/version-from-tag.py ${TAG} )
```

Comment:
Why not pure bash? ${TAG#v} removes prefix "v" from variable TAG

Reply:
Only because I forgot the syntax :) Fixing